src/gennylib/test/ActorHelper.hpp

```diff
@@ -84,6 +85,7 @@ class ActorHelper {
     std::unique_ptr<WorkloadContext> _wlc;
 
     std::stringstream _metricsOutput;
+    mongocxx::options::client _clientOpts;
```

Comment:
Why does the `ActorHelper` class need to store a copy of the `mongocxx::options::client` options?